README.md

```diff
@@ -32,7 +32,10 @@ If you have [installed node-gyp](https://github.com/nodejs/node-gyp#installation
 npm install -g @coveo/cli
 ```
 
-Afterwards, anytime you run `coveo update`, your CLI installation will also update to the latest version.
```

Comment:
We already say:

> These executables install the latest available version. Afterwards, anytime you run `coveo update`, your CLI installation will update to the latest version.

on line 27.

I'd rephrase the above slightly:

> Running one of the above executables will install the latest available version of the CLI.
>
> Afterward, you can run `coveo update` at any time to update your CLI installation to the latest version.

And here (i.e., under the npm installation instructions), I'd simply write:

> Afterward, you can run `npm update -g @coveo/cli` at any time to update your npm-based CLI installation to the latest version.

Reply:
Good idea!

Reply:
> We already say:
> 
> > These executables install the latest available version. Afterwards, anytime you run `coveo update`, your CLI installation will update to the latest version.
> 
> on line 27.
> 
> I'd rephrase the above slightly:
> 
> > Running one of the above executables will install the latest available version of the CLI.
> > Afterward, you can run `coveo update` at any time to update your CLI installation to the latest version.
> 
> And here (i.e., under the npm installation instructions), I'd simply write:
> 
> > Afterward, you can run `npm update -g @coveo/cli` at any time to update your npm-based CLI installation to the latest version.

@fbeaudoincoveo, following up on that, I restructured the readme a bit, wdyt?